src/GlobalUtils.ts

```diff
@@ -48,22 +49,27 @@ function speak(text): void {
 /**
  * Import a palette from the given json file using dynamic `import()`.
  *
- * Note:  There are restrictions regarding the arguments to `import()`:
- * - the path must start with "./" or "../" and not be part of the argument,
- * - the path must end with "/" and not be part of the argument,
- * - the file name extension must be added here (not part of the argument)
+ * Note:  The given JSON file is loaded as a string using 
  * See the following for more information:
- * https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
+ * https://vitejs.dev/guide/assets.html#importing-asset-as-string
  *
  * @param {String} jsonFile  - Name of the JSON file to load, without the
  *                            ".json" extension (added herein).
- * @param {String} path      - Path to the file to without any leading nor
- *                             trailing "/".
  * @return {JsonPaletteType} - The palette itself, or `null` if it could not be
  *                             loaded.
  */
-async function importPaletteFromJsonFile (jsonFile: string, path: string): Promise<JsonPaletteType> {
-  const paletteJson = await import(`./${path}/${jsonFile}.json`);
+async function importPaletteFromJsonFile (jsonFile: string): Promise<JsonPaletteType> {
+  // The "@vite-ignore" comment is used to suppress a warning generated by Vite during
+  // dynamic imports analysis. This warning is reported when both the file path and the
+  // filename are provided in a single variable. Using one sigle file location variable
+  // enhances the clarity and maintainability of the code, so this comment serves to
+  // mitigate the warning while ensuring the code remains sensible.
+  // See https://github.com/rollup/plugins/tree/master/packages/dynamic-import-vars#limitations
```

Comment:
The link to rollup's limitations appears twice.  It's also four lines down at line 71.

Is the link here possibly for documentation about `@vite-ignore`?